package/wifi-rtl8189fs/wifi-rtl8189fs.mk

```diff
@@ -1,5 +1,9 @@
 WIFI_RTL8189FS_SITE_METHOD = git
 WIFI_RTL8189FS_SITE = https://github.com/openipc/realtek-wlan
```

Comment:
please make this conditional with `else` so that it does not reassigns the value the second time.

Reply:
Fixed, thanks.